### ISSUE TITLE
Is scraping really required?

### ISSUE THREAD

**Issue by pranitbauva1997** [kossiitkgp/profile-generator#3]
Can't we just ask each and every member to send a PR to include their name on the website. Won't that make more sense, because then people can edit the content they want to write there?

And I hope all of our members do know how to send PRs, if they don't then maybe it's high time that they are kicked out of our team.

**Comment by americast** [comment]
@pranitbauva1997 The profile won't have just names. it would have the following:
1. Name
2. Profile Pic (github/fb/whatever)
3. Short Bio
4. most starred project , pinned repos
5. Link to personal website (if any)
6. Contact details
7. fb/github/twitter/blog links
All profiles would follow the same template.

**Comment by pranitbauva1997** [comment]
I think only these things are required:

1. Name
2. Profile Pic
3. Short Bio
4. Link to personal website
5. Github profile link (this should be a hyperlink to the name)

All other things are just unnecessary. The things that I have mentioned here can be done manually and actually, it should be done manually because I would want my bio to be different on every website to suit the purpose there.

**Comment by dibyadas** [comment]
@pranitbauva1997 
> Can't we just ask each and every member to send a PR to include their name on the website. Won't that make more sense, because then people can edit the content they want to write there?

That would have been a good option if not for the process one has to go through to do that. The person has to fork it and clone it and add his details file-wise and then commit and push to their fork and make a PR. Don't you think it's a bit too much for this?  We were planning to pass a google sheets with all the fields  and people can fill it up as they wish.  What is your take on this?

**Comment by pranitbauva1997** [comment]
@dibyadas If a person doesn't care enough to do this much, maybe he/she shouldn't exist at all in the team. This is a good way to know about the people we want to weed out.

**Comment by aribis369** [comment]
@pranitbauva1997 we were thinking of a members list page that would contain a short info and photo of that member and if users click on any of the members a separate page for that member will open having a details about the member and it's projects. It would be cumbersome to ask people to give so much details on theirs projects and activities. In my opinion it would be good to flaunt our member's work and their activities as people outside will be measure us by the projects we do and having such good force of programmers with superb project lineup would help us gain acknowledgement outside. It would be very kind of you to think about it and inform us what should be our next step.

**Comment by DefCon-007** [comment]
@aribis369 I don't think we need to brag about our personal projects on koss website. Those who will interested in someone's project can open up his/her GitHub profile. The things @pranitbauva1997 mentioned are enough I guess.

**Comment by Parth-Vader** [comment]
@DefCon-007 I'll back @dibyadas and @aribis369, having a page with projects and other information is okay in my opinion.

Dynamically updating it is also good. What's the harm?

**Comment by dibyadas** [comment]
@DefCon-007 I think what @pranitbauva1997 has mentioned will become too less to have an independent page for it. There won't be enough content.  Initially, we had decided to make our profile page dynamic,i,e, whenever someone's profile page is opened, all datas will be collected and then displayed. But then this was an unnecessary thing to do. So we decided to have it static. And this app was turned into a scraper. We will run this regularly and update the data. Refs issue #10

**Comment by DefCon-007** [comment]
@dibyadas I don't get why we need independent pages for members. Just link it the person's GitHub profile that's enough.
